testpmd-container-app/listener/Dockerfile

```
@@ -1,5 +1,19 @@
+FROM docker.io/library/golang:1.21 as build
+
+WORKDIR /scripts
+COPY scripts/webserver.go .
+RUN go mod init webserver.go
+RUN GOOS=linux CGO_ENABLED=0 go build -a -o webserver .
+
 FROM registry.access.redhat.com/ubi8/python-39:latest
 USER root
 RUN pip3 install kubernetes
+
+# Copy scripts
 COPY scripts /usr/local/bin
+COPY --from=build /scripts/webserver /usr/local/bin/
+
+# Update permissions
+RUN chmod +x /usr/local/bin/webserver
```

Avoid copying anything that is not needed in the image that is inside the scripts, e.g. the webserver.go

```suggestion
COPY --chmod=550 --from=build /scripts/webserver /usr/local/bin/
```